operations/security_and_authentication/Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md

```diff
@@ -0,0 +1,198 @@
+# Update default ServerTech PDU credentials used by the Redfish Translation Service
```

Comment:
Capitalization change: Update Default ServerTech PDU Credentials used by the Redfish Translation Service (RTS)"